### ISSUE TITLE
ts-standard parser error: the file does not match your project config

### ISSUE THREAD

**Issue by bmight** [standard/vscode-standard#156]
**What version of this package are you using?**

1.4.0

**What operating system, Node.js, and npm version?**

Windows 10, Node: v12.8.1, npm: 6.14.5

**What happened?**

I currently have a yarn workspace monorepo setup. When running ts-standard I get the following error for any file I open in my react project

```
Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: ..\..\..\..\..\..\c%3A\Users\[redacted]\packages\web\src\App.tsx.
The file must be included in at least one of the projects provided.
```

ts-standard seems to work fine on its own via CLI.

**What did you expect to happen?**

**Are you willing to submit a pull request to fix this bug?**

If I an understand what is causing it I'd be happy to help fix it.

**Comment by bmight** [comment]
This seems to be fixed in the release/v1.5.0 branch. I manual packaged it and installed it from that branch and things are working smoothly so far.

**Comment by microhobby** [comment]
I'm having the same issue, I can confirm that using the extension packaged from https://github.com/standard/vscode-standardjs/tree/release/v1.5.0 fix it.

Thanks @bmight
